maven-plugin/src/main/java/org/wildfly/galleon/maven/ArtifactListBuilder.java

```diff
@@ -96,6 +100,7 @@ private String checksum(String filepath) throws IOException {
     }
 
     public Path add(ArtifactCoords coords) throws ProvisioningException, ArtifactDescriptorException, IOException {
+        debug("Add artifact %s", coords.getGroupId() + ":" + coords.getArtifactId() + ":" + coords.getVersion());
```

Comment:
Combining a format with a string concatenation doesn't look good. Could you please change it to `%s:%s:%s`?

Reply:
Thanks @aloubyansky , fixed.